apiserver/plane/api/views/issue_type.py

```
@@ -0,0 +1,240 @@
+            # )
+            return Response(serializer.data, status=status.HTTP_201_CREATED)
+        return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
+
+    def patch(self, request, slug, project_id, issue_id, pk):
+        issue_type = IssueType.objects.get(
+            workspace__slug=slug,
+            pk=pk,
+        )
+        requested_data = json.dumps(self.request.data, cls=DjangoJSONEncoder)
+        current_instance = json.dumps(
+            IssueTypeSerializer(issue_comment).data,
```

Ruff (0.8.2)

Undefined name `issue_comment`

(F821)